src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/operations/AbstractOperations.java

```diff
@@ -148,7 +148,7 @@ protected void assertHasField(MessageOrBuilder messageOrBuilder, int... fields)
      * @see #getNextRange(SelectWhereStep, Field, Table, Object, boolean, int, Comparator)
```

Comment:
I dont know how we proceed with shitespace changes here - but they are very very useless and annoying.